internal/services/automation/automation_software_update_configuration_resource.go

```diff
@@ -628,7 +628,7 @@ func (m SoftwareUpdateConfigurationResource) Arguments() map[string]*pluginsdk.S
 								"occurrence": {
 									Type:         pluginsdk.TypeInt,
 									Required:     true,
-									ValidateFunc: validation.IntBetween(1, 5),
+									ValidateFunc: validation.IntInSlice([]int{1, 2, 3, 4, -1}),
```

Comment:
what currently happens if a user tries to set 5? does the API reject or acept it?

Reply:
API accepts `5`, but it will be displayed as empty on the Portal. If `-1` is set, it will be displayed as `Last` on the Portal.